go/service/user_handler.go

```diff
@@ -114,6 +114,7 @@ func (r *userHandler) userBlocked(m libkb.MetaContext, cli gregor1.IncomingInter
 			m.Warning("Error handling UserBlocked message: %s", tmp)
 		}
 	}
+	r.G().CardCache().Delete(msg.Uid)
```

Comment:
don't you want to delete badUIDs?

Reply:
@jzila, @patrickxb is correct

Reply:
I think this PR will work: https://github.com/keybase/client/pull/21430

If there's anything that it is missing that is in this one, let me know.